packages/browser/src/core/inspector/index.ts

```diff
@@ -20,12 +20,14 @@ const inspectorHost: Partial<InspectBroker> = ((env as any)[
 export const attachInspector = (analytics: Analytics) => {
   inspectorHost.attach?.(analytics as any)
 
-  analytics.on('dispatch_start', (ctx) => inspectorHost.triggered?.(ctx))
+  analytics.on('dispatch_start', (ctx) => inspectorHost.triggered?.(ctx as any))
 
-  analytics.queue.on('message_enriched', (ctx) => inspectorHost.enriched?.(ctx))
+  analytics.queue.on('message_enriched', (ctx) =>
+    inspectorHost.enriched?.(ctx as any)
```

Comment:
What changed that required these to be casted to `any` now?

Reply:
Previously, ctx was `any` here. Now, the `queue` [emitter is typed](https://github.com/segmentio/analytics-next/blob/dfb712f822c769eaef27b403f5743d9a02f96007/packages/core/src/queue/event-queue.ts#L13).

None of the context types line up with those of inspector, because we [don't share our Context type with inspector](https://github.com/segmentio/segment-inspector/blob/a175e0b37eab25676b27e5395c131b7ec6d14c23/packages/webext/src/broker/index.ts#L26). 

@zikaari's set the precedent for doing the coercion rather than say adding any mapping logic, so I just followed that pattern.

Reply:
Ah gotcha, makes sense!